src/lib/accessibility/Behaviors/Tree/treeTitleBehavior.ts

```diff
@@ -0,0 +1,15 @@
+import { Accessibility } from '../../types'
+
+/**
+ * @specification
+ * Adds attribute 'aria-expanded=true' based on the property 'open' and 'hasSubtree'
```

Comment:
Please what do you think if we change the line like:
Adds attribute 'aria-expanded=true' based on the property 'open' if the component has 'hasSubtree' property.

then it would invoke/consider more "expect" verification in the test itself... please see my comment in the unit tests for this behavior...

If you change this line, please don't forget to amend the regex expression as well...
Thank you :)